docs/src/docs/usage/install/index.mdx

```diff
@@ -53,6 +53,12 @@ brew install golangci/tap/golangci-lint
 brew upgrade golangci/tap/golangci-lint
 ```
 
+It can also be installed through [macport](https://www.macports.org/)
```

Comment:
I am not so sure if it's good point to mention macport here, as I don't think any one from @golangci/team will send PR to keep version in sync with latest release. 

Maybe just add some note saying that macport is community driven, and not official maintained by golangci team.

Reply:
Ok, I will indicate that this method is not officially supported by golangci-lint team

Reply:
@sayboras I've done the requested documentation change

Reply:
Thanks. LGTM :)